v24/src/main/java/com/foursoft/harness/kbl/v24/visitor/KblLocatedComponentVisitor.java

```diff
@@ -0,0 +1,469 @@
+/*-
+ * ========================LICENSE_START=================================
+ * kbl-v24
+ * %%
+ * Copyright (C) 2020 - 2022 4Soft GmbH
+ * %%
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ * 
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ * 
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ * =========================LICENSE_END==================================
+ */
+package com.foursoft.harness.kbl.v24.visitor;
+
+import com.foursoft.harness.kbl.v24.*;
+
+/**
+ * All methods are implemented as default method. Only methods which inherit from LocatedComponent are not overwritten.
+ */
+public interface KblLocatedComponentVisitor<R, E extends Throwable> extends Visitor<R, E> {

```

Comment:
@RainerGanss what do you think about this? do you think this is a good idea?

Reply:
No, I would not support such a use case specific visitor here in this lib. Don't we have the strict visitor here in the kbl lib? I'm afraid the burden of finding all classes inheriting a specific interface (here LocatedComponent) lies with the user of this lib. Otherwise we would have to create a visitor for every interface we have. How many do we support currently?